flake.nix

```diff
@@ -33,6 +33,15 @@
       url = "github:nix-community/flake-compat";
       flake = false;
     };
+    # for cabal-docspec
+    cabal-extras = {
+      url = "github:phadej/cabal-extras/cabal-docspec-0.0.0.20230517";
```

Comment:
I see that there is a [new version](https://github.com/phadej/cabal-extras/releases/tag/cabal-docspec-0.0.0.20231219), do we want to update it, or is this intentionally still using 20230517?